Dev_Tools_Install.md

```diff
@@ -18,19 +18,9 @@ Please install the following software in the exact order listed:
 
     <img src=".//media/image15.png"/>
 
-3. Download and run Microchip's [IoT Provisioning Tool](https://github.com/microchip-pic-avr-tools/iotprovision-bin/releases/latest) which is used to (1) generate the security certificates used for authentication and (2) store them into the [ATWINC1510](https://www.microchip.com/en-us/product/ATWINC1510) Wi-Fi controller module by executing the following steps:
+3. Install the Microsoft [Azure Command Line Interface (CLI)](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest). The Azure CLI is a set of commands used to create and manage Azure resources. The Azure CLI is available across Azure services and is designed to get you working quickly with Azure, with an emphasis on automation
```

Comment:
Remove language field from the link

https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest to

https://docs.microsoft.com/cli/azure/?view=azure-cli-latest

Reply:
Done